protocol/x/ratelimit/keeper/grpc_query.go

```diff
@@ -11,16 +12,38 @@ import (
 var _ types.QueryServer = Keeper{}
 
 func (k Keeper) ListLimitParams(
-	ctx context.Context,
+	goCtx context.Context,
 	req *types.ListLimitParamsRequest,
 ) (*types.ListLimitParamsResponse, error) {
-	// 	// TODO(CORE-823): implement query for `x/ratelimit`
-	return nil, status.Errorf(codes.Unimplemented, "method ListLimitParams not implemented")
+	if req == nil {
+		return nil, status.Error(codes.InvalidArgument, "invalid request")
+	}
+	ctx := sdk.UnwrapSDKContext(goCtx)
```

Comment:
looks like `sdkCtx := sdk.UnwrapSDKContext(ctx)` is more canonical variable naming in the codebase?